doc/examples/setup_assistant/setup_assistant_tutorial.rst

```diff
@@ -1,383 +1,520 @@
-:moveit1:
-
-..
-   Once updated for MoveIt 2, remove all lines above title (including this comment and :moveit1: tag)
-
 MoveIt Setup Assistant
 ========================
 
-.. image:: MoveIt_setup_assistant_launch.png
+.. image:: setup_assistant_launch.png
    :width: 700px
+   :align: center
 
 Overview
-----------------------
-The MoveIt Setup Assistant is a graphical user interface for
-configuring any robot for use with MoveIt. Its primary function is
-generating a Semantic Robot Description Format (SRDF) file for your
-robot. Additionally, it generates other necessary configuration files
-for use with the MoveIt pipeline. To learn more about the SRDF, you
-can go through the :doc:`URDF/SRDF Overview </doc/examples/urdf_srdf/urdf_srdf_tutorial>`
-page.
+--------
+The MoveIt Setup Assistant is a graphical user interface for configuring any robot for use with MoveIt.
+Its primary function is generating a Semantic Robot Description Format (SRDF) file for your robot,
+which specifies additional information required by MoveIt such as planning groups, end-effectors, and various kinematic parameters.
+Additionally, it generates other necessary configuration files for use with the MoveIt pipeline.
+To use the MoveIt Setup Assistant, you will need to have a URDF file for your robot.
+
+Once you have a URDF file, you can open the MoveIt Setup Assistant and import your URDF.
+This tutorial will guide you through a series of steps to configure various aspects of your robot,
+such as defining its kinematic structure, specifying planning groups and end-effectors, and collision checking related setting.
+To learn more about the URDF and SRDF, you can refer to the `URDF SRDF Overview <https://moveit.picknik.ai/humble/doc/examples/urdf_srdf/urdf_srdf_tutorial.html#>`_ page.
 
 Getting Started
-------------------------
+---------------
 
-MoveIt and ROS
+MoveIt and ROS 2
 
-* Follow the instructions for :moveit_website:`installing MoveIt<install>`
+* Follow the `instructions for installing MoveIt <https://moveit.picknik.ai/humble/doc/tutorials/getting_started/getting_started.html>`_
   first if you have not already done that.
 
-* If you haven't already done so, make sure you have the `Franka description
-  package <https://github.com/frankaemika/franka_ros>`_ for Noetic: ::
-
-    sudo apt install ros-noetic-franka-description
-
-* If you have the ``panda_moveit_config`` package already git-cloned from the *Getting Started* page, be sure to delete that now since this tutorial will teach you how to create it from scratch: ::
-
-   cd ~/ws_moveit/src
-   rm -rf panda_moveit_config
-   catkin clean panda_moveit_config
+* We use the `moveit_resources_panda_description <https://github.com/ros-planning/moveit_resources/tree/humble/panda_description/urdf>`_
+  package for Humble. This package should already be included in your workspace if you've completed the MoveIt installation instructions.
```

Comment:
For external links, you can keep the URL as is. However, try not to mention a specific release. I'd just point this to master and not mention Humble here.

Reply:
The tutorial is updated not to mention a specific release, but I think we should keep the link pointing to the Humble branch because the master is already changed. 

I made it like that because in the Humble release documentation, Getting Started Tutorial, [Download Source Code of MoveIt and the Tutorials](https://moveit.picknik.ai/humble/doc/tutorials/getting_started/getting_started.html#download-source-code-of-moveit-and-the-tutorials) section mentions the branch explicitly as follows.

```
cd ~/ws_moveit2/src
git clone https://github.com/ros-planning/moveit2_tutorials -b humble --depth 1
``` 
If we point to the master here, things will be different for the users. For example, `panda.urdf` is changed to `panda_arm_hand.urdf.xacro`.

I am aware of this point and will maintain the tutorial to keep things consistent. If you have any suggestions, please share with me and I will be happy to consider them.